### ISSUE TITLE
Atari - Jaguar CD

### ISSUE THREAD

**Issue by marimbaboy** [libretro-thumbnails/libretro-thumbnails#141]
Missing repository

**Comment by RobLoach** [comment]
We don't even have a database for it, seems like.
https://github.com/libretro/libretro-database/tree/master/rdb

I've added the definition file for it so next time we got through the database build process, at least we'll have a DB:
https://github.com/RobLoach/libretro-dats/commit/c8f6a1cd1bf615348b9bd287a570b670a70ddf73

There reallly aren't that many entries, and I'm unsure RetroArch has the serial discovery build in.

**Comment by marimbaboy** [comment]
You're right I hadn't noticed. Thanks for taking the time to look into it
